WrightTools/artists/_quick.py

```diff
@@ -148,7 +148,7 @@ def quick2D(
     autosave=False,
     save_directory=None,
     fname=None,
-    verbose=True
+    verbose=True,
```

Comment:
Extra comma is valid in py3.6, but not 3.5

Reply:
thanks for pointing this out--comma is now removed